src/java.base/share/classes/sun/security/ssl/SupportedGroupsExtension.java

```diff
@@ -112,7 +112,7 @@ private SupportedGroupsSpec(HandshakeContext hc,
         @Override
         public String toString() {
             MessageFormat messageFormat = new MessageFormat(
-                "\"versions\": '['{0}']'", Locale.ENGLISH);
+                "\"named groups\": '['{0}']'", Locale.ENGLISH);
```

Comment:
Per the TLS specification, it may be a more instinctive to use  "supported groups".

Reply:
> If so, should the `SupportedVersionsExtension` get a more precise description as well?

I did not get the point.  Did you mean to update the comment on the SupportedVersionsExtension, by adding RFC 8446?

Reply:
No, I just meant updating `"\"versions\": '['{0}']'"` to `"\"supported versions\": '['{0}']'"` there.

Reply:
I looked through SupportedGroupsExtension.java and found many examples, like the followings,
> no supported named group specified
the supported named groups
contains no supported named groups
No default named groups
Ignore inactive or disabled named group
... ...

so I finally used "named groups".